scripts/env.py

```diff
@@ -0,0 +1,18 @@
+import os
+
+
+class MissingEnvironmentVariableError(Exception):
+    def __init__(self, env_var):
+        return super().__init__(f"Missing environment variable {env_var}")
+
+
+def check_get_env(key):
+    """Return the value of the environment variable _key_.
+
+    Use this instead of os.getenv when the environment variable must be present
+    and if not an exception should be raised.
+    """
+    value = os.getenv(key)
+    if value is None:
```

Comment:
This is really minor, so feel free to disregard: I would suggest throwing a ValueError instead.

Could you also explain why we want to force environment variables to always be set? I could see an argument for not forcing it if you want to use default values that are overwritten by environment variables, but there are different opinions on default values, so curious to hear your rationale.

Reply:
`ValueError` doesn't seem to be the right candidate after looking at its documentation:

> Raised when an operation or function receives an argument that has the right type but an inappropriate value, and the situation is not described by a more precise exception such as IndexError.

@donnabelsey and I have gone through the list of all exception classes and it seems a custom exception type seems to be the best fit for our use-case.

Reply:
:+1:

Reply:
> Could you also explain why we want to force environment variables to always be set? I could see an argument for not forcing it if you want to use default values that are overwritten by environment variables, but there are different opinions on default values, so curious to hear your rationale.

So if this one particular environment variable isn't set, then the script breaks. But it breaks in a confusing way: there's no indication of what you'd need to do to fix it.

The only way to pass that data in is through the environment variable, and there isn't a default that would make sense, so that's why I wanted to make this change.